compiler-rt/lib/xray/xray_flags.h

```diff
@@ -17,6 +17,12 @@
 #include "sanitizer_common/sanitizer_flag_parser.h"
 #include "sanitizer_common/sanitizer_internal_defs.h"
 
+#define XRAY_OPTIONS_VAR __llvm_xray_options
```

Comment:
It doesn't seem useful to introduce a capitalized alias `XRAY_OPTIONS_VAR`. 

If a TU doesn't include `xray_flags.h`, the compiled code will reference `XRAY_OPTIONS_VAR`.